prov/sockets/src/sock_progress.c

```diff
@@ -64,6 +64,22 @@
 #define SOCK_GET_RX_ID(_addr, _bits) ((_bits) == 0) ? 0 : \
 	(((uint64_t)_addr) >> (64 - _bits))
 
+static inline int sock_pe_is_data_msg(int msg_id)
+{
+	switch (msg_id) {
+	case SOCK_OP_SEND:
+	case SOCK_OP_TSEND:
+	case SOCK_OP_WRITE:
+	case SOCK_OP_READ:
+	case SOCK_OP_ATOMIC:
+		return 1;
+	default:
+		goto out;
+	}
+out:
+	return 0;
```

Comment:
just have default call return 0, rather than jumping down one line then calling it